src/resources/OrganizationAccess/AccessInterfaces.ts

```diff
@@ -22,9 +22,32 @@ export interface AccessParams {
 }
 
 export interface AccessModel {
```

Comment:
I wonder if we would better off splitting this interface in two, because callerPartOf only exists when privilegeHolderType = groups. You cannot be "part of" an api key. Also `createdDate` does not exist for Groups

I'm thinking about something like:

```ts
type ApiKeyAccessModel = {
    accessLevel: AccessLevel;
    createdDate?: string;
    displayName: string;
    id: string;
    privilegeHolderType: PrivilegeHolderType.API_KEY;
}

type GroupAccessModel = {
    accessLevel: AccessLevel;
    callerPartOf?: boolean;
    displayName: string;
    id: string;
    privilegeHolderType: PrivilegeHolderType.GROUP;
}

export type AccessModel = ApiKeyAccessModel | GroupAccessModel;
```

By doing this we could also be more precise in the jsdocs about what the displayName and id represents.